packages/base/Slider/src/Slider/story/Tooltip.example.tsx

```diff
@@ -2,8 +2,12 @@ import React from 'react'
 import { Container, Slider, Typography } from '@toptal/picasso'
 import { SPACING_4, SPACING_8, SPACING_10 } from '@toptal/picasso-utils'
 
-const formatLabel = (val: any) => {
-  const formattedVal = val.length === 2 ? val : '0' + val
+type Value = number | number[]
+
+const formatLabel = (value: Value) => {
+  let formattedVal = String(value)
+
+  formattedVal = formattedVal.length === 2 ? formattedVal : '0' + value
```

Comment:
We have the JS [`.padStart`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) function, which implements this logic
```suggestion
  const formattedVal = String(value).padStart(2, '0')
```

Also, why are we changing the `value` argument to an array? Any chance we get an array on this line and we do a `String(value)` on it? On that case we might have a few commas on the output and mess the padding

Reply:
I only made this change because the old implementation had a bug where 0 was added even for two digit numbers. So I took the snippet code from the other stories where it was working correctly. But yeah, I don't see any chance of having an array, so I'll remove the type variant from Value.